session-02/exercise.js

```diff
@@ -4,6 +4,8 @@
 */
 function transformArrayToUpper(listOfStrings) {
   return;
```

Comment:
Your tests are failing because this empty return statement needs to be removed. Your code wont execute because this ends the function first.